collect_app/src/main/java/org/odk/collect/android/activities/InstanceChooserList.java

```diff
@@ -213,6 +213,8 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                             c.getString(c.getColumnIndex(DatabaseInstanceColumns.CAN_EDIT_WHEN_COMPLETE));
 
                     boolean canEdit = status.equals(Instance.STATUS_INCOMPLETE)
+                            || status.equals(Instance.STATUS_INVALID)
```

Comment:
Was there a reproducible problem here? It feels like we were probably missing a test if so.

Reply:
Ah sorry I should have put that together. So I guess instead of adding the new statuses we could just remove the check on `canEdit` (and the logic to assign it) right? It seems like it isn't currently needed.

Reply:
No, I said strCanEditWhenComplete is always true in the list of drafts but this activity is also used to display the list of sent forms and then if a sent form was encrypted we shouldn't allow to open it. In such a case strCanEditWhenComplete will be false.

Reply:
Ah I keep forgetting `InstanceChooserList` is used in both places. It looks to me like we can still delete that code though - we have a couple of other checks that take care of the encrypted scenario ([here](https://github.com/getodk/collect/blob/da182e47189f3b599cfbc0e6cd82c9cded56dbe0/collect_app/src/main/java/org/odk/collect/android/activities/InstanceChooserList.java#L239) and [here](https://github.com/getodk/collect/blob/da182e47189f3b599cfbc0e6cd82c9cded56dbe0/collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt#L194)). These are both covered by tests, but I can remove the `canEdit` code and nothing fails. The only other place I thought it might apply was for partial submissions, but the `FormUriActivity` check currently takes care of that.

Reply:
Right, I've removed that code.